### ISSUE TITLE
Webhooks don't do anything?

### ISSUE THREAD

**Issue by parkerwebteam** [Worldpay/worldpay-woocommerce#16]
Hello,

It would appear that the configuration settings are slightly out of date with the WorldPay interface. For example, the webhook URL that is given within the WorldPay settings area is http:// and the WorldPay Online interface forces you to use https://.

With that aside, manually entering https:// doesn't seem to work at all, i.e. The Webhooks do not reach the server or update any Woocommerce Orders at all.

Is this project actively maintained at all, is there any support process for this?

**Comment by dion1512** [comment]
@parkerwebteam did you ever get anywhere with this? we are having the same problems with webhooks and WorldPay support have been less than helpful

**Comment by parkerwebteam** [comment]
@dion1512 We never received a response in regard to this from WorldPay - we submitted a traditional ticket to their support channel and did not receive any response. As a solution, we ended up going with a merchant account using Woocommerce's WorldPay solution. 

TL;DR Find an alternative solution.
